pkg/occlient/occlient.go

```diff
@@ -297,7 +298,12 @@ func (c *Client) CreateNewProject(name string) error {
 	return nil
 }
 
-func (c *Client) SetCurrentProject(project string) error {
+// SetCurrentProject sets the passed project as active in kube config file
+// Parameters:
+//	1. project to set as active
+//	2. the instance of client to be used to set the project as active
+// Returns: error. non nil error in case of errors, nil otherwise
+var SetCurrentProject = func(project string, c *Client) error {
```

Comment:
Why have you done like this? It breaks the pattern that we have been using everywhere :-(

Reply:
Only to mute it during testing as in https://github.com/redhat-developer/odo/pull/740/files#diff-c6cce71dcf9031800c52b798ff473d1cR38.

Will try to find a better way to achieve this...

Reply:
Could it be done in FakeClient https://github.com/redhat-developer/odo/blob/master/pkg/occlient/fakeclient.go ?

Reply:
Not FakeClient because it is only file operations..
Now changed it to mock file operations as well